tests/PhpSpreadsheetTests/Functional/HtmlCommentsTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace PhpOffice\PhpSpreadsheetTests\Functional;
+
+use PhpOffice\PhpSpreadsheet\Spreadsheet;
+
+class HtmlCommentsTest extends AbstractFunctional
+{
+    private $value = 'I am comment.';
+    private $spreadsheet;
+
+    private function createComment()
+    {
+        $this->spreadsheet = new Spreadsheet();
+
+        $this->spreadsheet->getActiveSheet()->getCell('A1')->setValue('Comment');
+
+        return $this->spreadsheet->getActiveSheet()
+            ->getComment('A1')
+            ->getText()->createTextRun($this->value);
+    }
+
+    private function doTheTest($string)
+    {
+        $reloadedSpreadsheet = $this->writeAndReload($this->spreadsheet, 'Html');
+
+        $actual = (string) $reloadedSpreadsheet->getActiveSheet()->getComment('A1')->getText()->getPlainText();
```

Comment:
`(string)` should not be needed here, since `getPlainText()` already returns a `string`

Reply:
Hehe, that's an leftover before I used `getPlainText()` ^^